### PR TITLE
feat: displayed loans event implemented for new featured loan test

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- displayed loans event implemented for new featured loan test

### CODE REVIEW

File: src/pages/LoanFinding/LoanFinding.vue [hunk 1]
```diff
@@ -366,7 +366,7 @@ export default {
 			}
 		},
 		trackFirstRowDisplayedLoans() {
-			this.trackDisplayedLoans(this.enableRelendingExp ? 'relending' : 'recommended', 1, this.firstRowLoans);
+			this.trackDisplayedLoans(this.enableRelendingExp && !this.enableRecommendedReplacementExp ? 'relending' : 'recommended', 1, this.enableRecommendedReplacementExp ? [this.featuredLoan] : this.firstRowLoans); // eslint-disable-line max-len
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:369]: We will be removing the relending experiment, so you could simplify this line now.

File: src/pages/LoanFinding/LoanFinding.vue [hunk 2]
```diff
@@ -366,7 +366,7 @@ export default {
 			}
 		},
 		trackFirstRowDisplayedLoans() {
-			this.trackDisplayedLoans(this.enableRelendingExp ? 'relending' : 'recommended', 1, this.firstRowLoans);
+			this.trackDisplayedLoans('recommended', 1, this.enableRecommendedReplacementExp ? [this.featuredLoan] : this.firstRowLoans); // eslint-disable-line max-len
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:369]: It actually looks like we also need a fixed fallback here: https://github.com/kiva/ui/blob/048d035a642cc44bb482cdc050104b52390c05c5/src/pages/LoanFinding/LoanFinding.vue#L449

`})?.fundraisingLoans?.values[0] ?? [{ id: 0 }];` -> notice the array

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:369]: Well, with `fundraisingLoans?.values[0]` we are just returning the loan object and assigning to this `featuredLoan: {},`.
Said that, we don't need the array.
If we  want to preserve the array in both, we need to update the assignment to the `featureLoan` var.

Comment (dyersituations) [reply src/pages/LoanFinding/LoanFinding.vue:369]: Oh yup, you're right.

**Pull request merged**